core/controllers/editor.py

```diff
@@ -67,7 +67,11 @@ class ExplorationPage(EditorHandler):
 
     @acl_decorators.can_play_exploration
     def get(self, unused_exploration_id):
-        """Handles GET requests."""
+        """Handles GET requests.
+
+        Args:
+            unused_exploration_id: Gets the data for the exploration overview page. (Unused).
```

Comment:
Rephrase it to 'exploration editor page'? Since we don't use the term 'exploration overview' anywhere, I think.

Reply:
Arg type also missing.